lib/util/connection.ts

```diff
@@ -51,6 +55,19 @@ export class Connection extends AsyncEventEmitter<ConnectionEvents> {
   private packetGroupReliability: PacketType = PacketType.ReliablePacket;
   unacknowledgedPackets: Map<number, number> = new Map();
   private TEMPDONTUSE: boolean = false;
+  private static internalSupportedVersions: ClientVersion[] = [
```

Comment:
Not sure this is the right place to store these, but I'm also not sure we have anywhere better just yet. Maybe we should introduce a `config.ts` file or something?

Reply:
I would have put them on the server but there's no immediate way to get those versions from within the connection. I thought about emitting a `hello` event from the connection but that seemed a little odd for a version check.

Reply:
there is a server `connection` event that acts on a `hello` packet

Reply:
What do these versions have to do with a connection? They're just initd when the server starts right? We can just store them as a singleton somewhere that can be imported where needed

Reply:
Well a connection/server is implementing the protocol which has specific supported versions, which is why I thought to just make it a property on one of the two. It could be separated into its own file if people would prefer that.

Reply:
They could even go on the `ClientVersion` class as a static property